tasks/dockerCompose.js

```diff
@@ -150,14 +162,11 @@ module.exports = function(grunt) {
 
 		// only do this AFTER setting the tag. Otherwise it won't be merged into the command.
 		var cmd = buildCommandSkeleton();
-
 		cmd.push('docker-compose');
-		if (!grunt.option('baked') && !grunt.option('debug')) {
+		if (grunt.option('map') && !grunt.option('baked') && !grunt.option('debug')) {
```

Comment:
so if we don't specify `--map`, this effectively amounts to `--baked`, as we're not adding the `-f` option to the command.  Using the mapped file should remain as default.